web/ui/two-steps-page.svelte

```diff
@@ -23,25 +27,38 @@
     })
   }
 
+  function handleEscapeKey(event: KeyboardEvent): void {
```

Comment:
The current flow works only if you use mouse before only then press `Esc`.

But if you use keyboard, it will not work as expected.

Try this flow:
1. Press <kbd>m</kbd>
2. Press <kbd>↵</kbd> (to open Add feed)
3. Type `github.blog`
4. Press <kbd>↵</kbd> (to move to found items)
5. Press <kbd>↵</kbd> (to move to side panel)
6. Press <kbd>Esc</kbd>. Expected: move back to search form and close side panel. Actual behavior: panel is still open.